src/rustdoc/proposals-and-stabilization.md

```diff
@@ -0,0 +1,70 @@
+# Proposals, Approvals and Stabilization
+It is very common to need to gather feedback and approval when contributing to rustdoc, either
+for permission to proceed with an experiment or refactoring, or when stabilizing a feature. This
+document aims to summarise the various processes that the rustdoc team has for making approval
+decisions and when each should be used.
+
+## Approvals
+There are three mechanisms that the team can use to approve a proposal (not all approval mechanisms
+are suitable for each method of making a proposal - see below):
+
+- r+
+  - A proposal is r+'d when it is approved to be merged.
+  - r+ can only be used to approve a PR.
+- FCP
+  - A final comment period will require sign-off from a majority of the rustdoc team to approve
+    a proposal and then a ten day waiting period.
+  - FCPs can be used to approve any form of proposal.
+
+## Proposals
+There are two ways to propose a change to the rustdoc team. The appropriate choice depends on
+the nature of the proposal, described below.
+
+- Request For Comments (RFC)
+  - RFCs are pull requests to the [`rust-lang/rfcs`][rfcs] repository and are a heavy-weight
+    proposal mechanism, reserved for significant changes.
+  - RFC proposals can only be approved by *FCPs*.
+- Pull Request (PR)
+  - PRs are pull requests to the [`rust-lang/rust`][rust] repository and are a light-weight
+    proposal mechanism, suitable for most proposals. PRs are preferred when the proposal is
+    accompanied by a small patchset (such as stabilization of a compiler flag or addition of
+    a new target).
+  - PR proposals can be approved by *FCPs* or *r+*.
+
+### When are FCPs/RFCs required?
+
+An FCP will be needed for any stabilization of small user-facing changes, like UI/UX changes, new
+command-line arguments, new attributes, etc. However, if the change is considered too big/important,
```

Comment:
We should mention insta-stable GUI changes and the relaxed stability requirements thereof, since those aren't really a thing that exist anywhere else in rust project software.

Reply:
`rustdoc` GUI is not considered stable (like the HTML/JS/CSS). We just try to limit its changes as much as possible. If it is a user-facing change, we want to go through an FCP to ensure that the reasons behind the change are valid and that majority agrees.

Reply:
I think it would be good to clarify that UI/UX refers to the programmatic interface of rustdoc, not the web GUI (except for major changes).

Reply:
No, user-facing changes, like UI/UX in the rustdoc GUI web interface are definitely part of the thing we want to consider carefully and thus go through FCP, even if this interface is not considered stable.